src/Session.php

```diff
@@ -55,10 +55,12 @@ public function isStarted()
     }
 
     /**
-     * Starts session driver.
+     * Starts session driver if session is not started.
      *
      * Calling any action before visiting a page is an undefined behavior.
      * The only supported method calls on a fresh driver are
+     * - resizeWindow()
+     * - maximizeWindow()
```

Comment:
Correct, that is my bad. Where would you add the tests? Looking at the CoreDriverTest and the Tests in the Selenium driver I could not see any tests on e.g. the visit method